specs/core/1_custody-game.md

```diff
@@ -212,44 +212,50 @@ Add the following fields to the end of the specified container objects. Fields w
 #### `Validator`
 
 ```python
-    # next_custody_reveal_period is initialized to the custody period
+{
+    # next_custody_reveal_period is initialised to the custody period
     # (of the particular validator) in which the validator is activated
     # = get_validators_custody_reveal_period(...)
     'next_custody_reveal_period': 'uint64',
     'max_reveal_lateness': 'uint64',
+}
 ```
 
 #### `BeaconState`
 
 ```python
+{
     'custody_chunk_challenge_records': [CustodyChunkChallengeRecord],
     'custody_bit_challenge_records': [CustodyBitChallengeRecord],
     'custody_challenge_index': 'uint64',
 
     # Future derived secrets already exposed; contains the indices of the exposed validator
     # at RANDAO reveal period % EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS
     'exposed_derived_secrets': [['uint64'], EARLY_DERIVED_SECRET_PENALTY_MAX_FUTURE_EPOCHS],
+}
 ```
 
 #### `BeaconBlockBody`
 
 ```python
+{
     'custody_chunk_challenges': [CustodyChunkChallenge],
     'custody_bit_challenges': [CustodyBitChallenge],
     'custody_responses': [CustodyResponse],
     'custody_key_reveals': [CustodyKeyReveal],
     'early_derived_secret_reveals': [EarlyDerivedSecretReveal],
+}
 ```
 
 ## Helpers
 
-### `typeof`
+### `type_of`
```

Comment:
Do we want to call this `typeof` or the more pythonic `type-of`?

Reply:
`type_of` is easier to read. Could we also just use `type` and not need this definition at all? I guess this implicitly depends on how SSZ is implemented in python then?

Reply:
See my other comment below -- seems like we could just use python `type()` here, right? What's the argument against doing that?